README.md

```diff
@@ -92,6 +92,14 @@ Run tests with:
 pnpm run test
 ```
 
+Update snapshots with:
+
+```
+pnpm build
+cd packages/big-design
+npx jest --update-snapshot
```

Comment:
Since our docs reference pnpm, let's use that instead of npm (I believe the `--` is needed but not entirely confident).
```suggestion
pnpm run test -- --update-snapshot
```

Reply:
Tested (the middle `--` was unnecessary) and updated.